vmdb/app/models/miq_event_catcher_openstack_infra.rb

```diff
@@ -0,0 +1,18 @@
+class MiqEventCatcherOpenstackInfra < MiqEventCatcher
```

Comment:
The only difference there would be that validate_config_settings would also inherit from EventCatcherOpenstack -- from the look of the method, I *think* that's OK -- in any case this change either fixes a bug in my code or causes one, depending on whether that's correct.